CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2021-04-01 Ability to Search in Single Choice Drop-Down Lists
+<p>Single Choice Drop-Downs are now editable, which would filter the options in the drop down list based on what is typed.</p>
```

Comment:
change "editable" to "searchable" and "which would" to "which allows you to"

Reply:
Yes they are not editable. Fixed.